package.json

```diff
@@ -2,48 +2,48 @@
   "name": "laboratorium-ee-2019",
   "version": "0.0.1",
   "private": true,
+  "engines": {
+    "node": "16.x"
+  },
```

Comment:
Chyba mieliśmy się na razie trzymać LTS :p
```suggestion
  "engines": {
    "node": "14.x"
  },
```

Reply:
od niedawna lts to 16

Reply:
O kurde, to trzeba przenieść do generatorów